demisto/client.py

```diff
@@ -11,6 +11,7 @@
 from requests import Session
 from requests.packages.urllib3.exceptions import InsecureRequestWarning
 from requests.packages.urllib3 import disable_warnings
+import ntpath
```

Comment:
please use `os.path` and not `ntpath` as it is generic

Reply:
done